Cargo.toml

```diff
@@ -32,7 +32,7 @@ log = "0.4.17"
 # Note that `tame-index` and `gix` must be upgraded in lock-step to retain the same `gix`
 # minor version. Otherwise, one will compile `gix` two times in different minor versions.
 gix = { version = "0.63", default-features = false, features = ["max-performance", "revision"] }
-tame-index = { version = "0.12", features = ["git", "sparse"] }
```

Comment:
Doesn't this PR end up removing the `git` feature that we depend on to support gitrev baselines?

Specifically, this CLI flag: ([docs](https://github.com/obi1kenobi/cargo-semver-checks?tab=readme-ov-file#does-the-crate-im-checking-have-to-be-published-on-cratesio))
```
--baseline-rev <REV>
    Git revision to lookup for a baseline
```

Reply:
I checked the features here, and IIUC it should still be enabled by default, but will test it locally just in case:  
https://github.com/EmbarkStudios/tame-index/blob/9bc174aebc946d87dbb5a68ee88ccc21b6d2bc69/Cargo.toml#L15-L20

Reply:
I just checked, and our CI already has a test for `--baseline-rev` which passed successfully.

I'm pushing a couple of minor fixes to your branch, and will then merge the PR & release in a new version shortly. Thanks for the PR, and if you're able to offer sponsorship funding, I'd really appreciate it!